planning/obstacle_cruise_planner/include/obstacle_cruise_planner/planner_interface.hpp

```diff
@@ -235,16 +266,37 @@ class PlannerInterface
         node.declare_parameter<double>("slow_down.lpf_gain_dist_to_slow_down");
     }
 
+    ObstacleSpecificParams getObstacleParamByLabel(const ObjectClassification & label_id) const
+    {
+      std::string label = types_map.at(label_id.label);
```

Comment:
```suggestion
      const std::string label = types_map.at(label_id.label);
```

Reply:
Thanks for the suggestions @maxime-clem I have added them to the code.